lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -36,7 +36,14 @@ const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
       const largeArc = percent > 50 ? 1 : 0;
 
       return (
-        <svg ref={ref} viewBox="-15 -15 30 30" role="img" className={donutClasses} {...svgAttrs}>
+        <svg ref={ref}
+             viewBox="-15 -15 30 30"
+             role="meter"
+             aria-valuemin={0}
+             aria-valuemax={100}
```

Comment:
I wonder if we should introduce a new optional prop to allow calling code to set a different max.  Oftentimes the donut chart is used to represent some count out of some total, such as "87 out or 95 components identified".  It'd be a shame to lose those real numbers for the accessibility metadata.

Reply:
Agreed. I will add an optional prop for the different maximum value .

Reply:
Optional prop added, thanks.

Reply:
> Optional prop added, thanks.

Don't forget to reference the commit where you've fixed something when you respond saying that it is fixed

Reply:
Got it, the commit for adding optional prop is here 1dacc26140b8acee4ca38610eb64886a5b2821be, thanks.